services/movie.py

```diff
@@ -15,13 +17,16 @@
     if actors_ids:
         queryset = queryset.filter(actors__id__in=actors_ids)
 
+    if title:
+        queryset = queryset.filter(title__icontains=title)
     return queryset
 
 
 def get_movie_by_id(movie_id: int) -> Movie:
     return Movie.objects.get(id=movie_id)
```

Comment:
Consider adding error handling for the `get_movie_by_id` function to manage cases where a movie with the given ID does not exist. This can prevent the function from raising a `DoesNotExist` exception.